pkg/application/application.go

```diff
@@ -201,3 +183,17 @@ func SetCurrent(name string) error {
 
 	return nil
 }
+
+func Exists(name string) error {
```

Comment:
shouldn't this also return bool?

It is not possible to determine if application doesn't exist or if there were error while checking it

Reply:
correct, it should return both

Reply:
fixed